### DISCUSSION TITLE
Quickly change server connection

### DISCUSSION THREAD

**Discussion by woodyst** [rustdesk/rustdesk#5178] [Feature Request]
I am an IT consultor and usually use rustdesk with different owned hbbs, so I have to change server settings in my rustdesk client every time I change the connection to a different server.

I have been trying to make a script to change client's server settings in a fast way, launching the script, but I haven't been able to get it.

My os is Debian GNU/Linux and I have seen rustdesk settings in .config/rustdesk directory, and server settings in RustDesk2.toml file. I have tried to copy the file to RustDesk2.toml.server1, ...server2, etc and copy content of the copied file to RustDesk2.toml before launching rustdesk, but settings do not change in gui. I also have tried to kill all rustdesk processes before copying files and then launch client, but no results again.

Please, can you explain me how to do this task from a shell script?

I would be very interested also in  implementing a server list in the client, saving all preferences per server in the gui and to be able to select the server with a drop down list or something similar.

Thanks in advance.

Edu.

**Comment by SergeyMy** [comment]
remove rustdesk from the system (or termination of service) and use it as portable = user settings will work, not root.
PS I doubt that clients need to connect to you :)

**Comment by goodluck1982** [comment]
>I would be very interested also in implementing a server list in the client, saving all preferences per server in the gui and to be able to select the server with a drop down list or something similar.

I also think it is an excellent idea to  implement a server list in the client for switching conveniently.

**Reply by goodluck1982** [reply]
And at best user can specify a server for each peer.

**Comment by rustdesk** [comment]
[[FEATURE] - Connect even if device are not on same server #6198
](https://github.com/rustdesk/rustdesk/issues/6198)
